src/overmapbuffer.cpp

```diff
@@ -486,6 +486,27 @@ void overmapbuffer::signal_hordes( const tripoint_abs_sm &center, const int sig_
     }
 }
 
+void overmapbuffer::signal_nemesis( const tripoint_abs_sm &p, const int sig_power )
+{
+
+    const auto radius = sig_power;
+    for( auto &om : get_overmaps_near( p, radius ) ) {
```

Comment:
Presumably this auto here should be overmap?

Reply:
i'm actually struggling with figuring out what this auto should be; i will admit here to just having copied the loop code from the existing "signal_hordes" function, knowing that it serves my purposes but not precisely how it works.

Reply:
nevermind! i figured this out 😄